grease-cli/tests/sanity/pass/dwarf-shapes/test.c

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) Galois, Inc. 2025 */
+
+// Tests that heuristics are unneeded when using dwarf populated shapes to 
+// create a memory precondition that allows foo to execute.
+
+struct bar {
+  int x;
+};
+
+int foo(int x, int *y, struct bar *z) { return x + *y + z->x; }
+
+// all: flags {"--symbol", "foo", "--enable-dwarf-preconditions", "--no-heuristics"}
+// x64: go(prog)
+// x64: ok()
```

Comment:
Most of our tests use the entrypoint `test`. Since `foo` isn't a significant name, perhaps let's use `test` here too